daemon/cmd/daemon_main.go

```diff
@@ -1314,11 +1315,16 @@ func initEnv(vp *viper.Viper) {
 	bpf.CheckOrMountFS(option.Config.BPFRoot)
 	cgroups.CheckOrMountCgrpFS(option.Config.CGroupRoot)
 
+	monitorEvents := option.Config.HubbleMonitorEvents
+	var enableEventGeneration = func(eventType string) bool {
```

Comment:
Should we default to true if hubble is disabled?

Reply:
@marqc please resolve this discussion if you have adressed it in your current PR. The discussion is blocking the merge.